src/config/dark-sites.config

```diff
@@ -231,6 +231,8 @@ disneyplus.com
 distrotube.com
 dlive.tv
 dodi-repacks.site
+dood.la
```

Comment:
We don't allow sites that are used as redirects.

Reply:
Wait, let me guess... dood.la is used as "shorten link"?

Reply:
Sample:
it is video stream file on doodstream.com 
Instead of long `doodstream.com/path/foo/bar.mp4`
You have link dood.la/s/abscdfgr

This is same server provider, using own domain for share link to content and embed player. 

Against DMCA I cannot share full link for valid content.

Reply:
Got it, seems good then.